system-parachains/asset-hubs/asset-hub-polkadot/primitives/src/lib.rs

```diff
@@ -52,7 +52,7 @@ frame_support::parameter_types! {
 	pub UncongestedMessage: Xcm<()> = build_congestion_message(false).into();
 
 	/// Should match the `AssetDeposit` of the `ForeignAssets` pallet on Asset Hub.
-	pub const CreateForeignAssetDeposit: u128 = 10 * UNITS;
+	pub const CreateForeignAssetDeposit: u128 = system_para_deposit(1, 190);
```

Comment:
cc @claravanstaden @acatangiu

Reply:
190 is based on the bytes? there should be some way to get that number without hardcoding no?

with `max_encoded_len`?

Reply:
It is the encoded length of [`AssetDetails`](https://github.com/paritytech/polkadot-sdk/blob/bc2e5e1/substrate/frame/assets/src/types.rs#L52) in this configuration.

I'm not sure if we can use `max_encoded_len` here? I calculated it by writing a test and asserting `encoded_len`.

Reply:
If you remove the const it should work.